src/sentry_plugins/asana/plugin.py

```diff
@@ -146,31 +147,31 @@ def error_message_from_json(self, data):
             return " ".join(e["message"] for e in errors)
         return "unknown error"
 
-    def create_issue(self, request, group, form_data, **kwargs):
+    def create_issue(self, request: Request, group, form_data, **kwargs):
         client = self.get_client(request.user)
 
         try:
             response = client.create_issue(
                 workspace=self.get_option("workspace", group.project), data=form_data
             )
         except Exception as e:
-            self.raise_error(e, identity=client.auth)
+            raise self.raise_error(e, identity=client.auth)
```

Comment:
@mgaeta self.raise_error raises the error itself. I don't think we need this `raise` here.

Reply:
pyright doesn't understand the `NoReturn` return type so its flow analysis needs this hint to avoid the `reportUnboundVariable` error.

Reply:
NIT: In that case, I think `return` makes more sense, since the error is raised by the function.